tests via GitHub Actions

mr-tz opened the pull request:
- use GitHub Actions for code style and tests
- use `flare-floss-testfiles` submodule
  - includes compiled binaries and source
- auto-adjusted code style
  - isort
  - black

mr-tz committed:
tests via GitHub Actions

mr-tz committed:
fix isort

mr-tz committed:
fix black code style

mr-tz committed:
move src to data dir

mr-tz committed:
remove data dir

mr-tz committed:
add test data as submodule

mr-tz committed:
checkout with submodules

mr-tz committed:
update docs

williballenthin commented on README.md:
```diff
@@ -25,14 +24,14 @@ Please review the theory behind FLOSS [here](doc/theory.md).
+  - Windows 64bit: TODO
+  - Windows 32bit: TODO
+  - Linux: TODO
+  - OSX: TODO
```
i think we could maybe just remove this in favor of small point releases

mr-tz replied:
removed

williballenthin reviewed (approved):
i love this!

jedimasterbot committed:
Update strings.py

Fix for the issue  #347, converting all the regex into bytes based and the issue is resolved

https://github.com/fireeye/flare-floss/issues/347

williballenthin committed:
Merge pull request #348 from jedimasterbot/patch-1

Update strings.py

williballenthin committed:
Merge branch 'master' into ci/tests
